lightly_studio/src/lightly_studio/core/loading_log.py

```diff
@@ -42,13 +45,14 @@ def log_loading_results(
     """
     n_samples_end = sample_resolver.count_by_dataset_id(session=session, dataset_id=dataset_id)
     n_samples_inserted = n_samples_end - logging_context.n_samples_before_loading
-    print(
-        f"Added {n_samples_inserted} out of {logging_context.n_samples_to_be_inserted}"
-        " new samples to the dataset."
+    logger.info(
+        f"Added {n_samples_inserted} out of {logging_context.n_samples_to_be_inserted} "
+        "new samples to the dataset.",
     )
     if logging_context.example_paths_not_inserted:
-        # TODO(Jonas, 09/2025): Use logging instead of print
-        print(
-            f"Examples of paths that were not added: "
-            f" {', '.join(logging_context.example_paths_not_inserted)}"
+        logger.warning(
+            (
+                "Examples of paths that were not added: "
+                f"{', '.join(logging_context.example_paths_not_inserted)}"
+            ),
         )
```

Comment:
I'm a bit surprised that we are passing in a one-element string tuple. Does not just passing a string work?
```suggestion
    logger.info(
        f"Added {n_samples_inserted} out of {logging_context.n_samples_to_be_inserted} "
        "new samples to the dataset."
    )
    if logging_context.example_paths_not_inserted:
        logger.warning(
            "Examples of paths that were not added: "
            f"{', '.join(logging_context.example_paths_not_inserted)}"
        )
```